source/user-manual/capabilities/vulnerability-detection/configuring-scans.rst

```diff
@@ -55,6 +55,13 @@ The following configuration block shows a configuration example for Syscollector
       </synchronization>
     </wodle>
 
+Make sure you have saved the Wazuh indexer username and password into the Wazuh manager keystore using the Wazuh-keystore tool.
```

Comment:
```suggestion
You must save the Wazuh indexer username and password into the Wazuh manager keystore using the Wazuh-keystore tool.
```

Reply:
There is no precedence or mandatory order, as long as both changes are applied before the Manager is restarted